### ISSUE TITLE
Separate audio editing, playback, and saving

### ISSUE THREAD

**Issue by LeedsJohn** [WhitmanCS370/Johns-Frends#86]
I don't really like the interaction of one command to apply edits to a sound, play it, and optionally save it.

I personally think that a better option would be to introduce an edit command so you could run `edit [sound] [new name for edited sound] [list of audio effects]` (or something like that), and then we would remove all the audio effects from the play command.

Or, alternatively, we could still leave the audio effects in the play command, but I don't know if I think you should be able to save a new sound from the play command.

@Rhys-sg @samueluk02 Let me know what you guys think about this

**Comment by samuelsl02** [comment]
I agree that it feels a little bloated to have playing, editing, and saving all in the same command but I really like that we can edit/play a sound without saving it. I think if we refactored this, it would feel best (to the user) if an edit command had an option to play the sound after editing - it would look like `edit [sound] [list of effects] [--play_sound (optional)][--save (optional)][new name]`. If --save isn't used we could just play it.

**Comment by LeedsJohn** [comment]
i like this idea!

**Comment by LeedsJohn** [comment]
I'm going to close #44 because it's similar to this one but wanted to leave a link to it here because @Rhys-sg left some good ideas in his issue.
